plugins/modules/dcnm_vrf.py

```diff
@@ -33,6 +33,8 @@
 version_added: "0.9.0"
 description:
     - "Add and remove VRFs and VRF Lite Extension from a DCNM managed VXLAN fabric."
+    - "In Multisite fabrics, VRFs can be created only on Multisite fabric"
```

Comment:
I can't remember, but if you attempt to create a vrf on the member fabric does DCNM generate a meaningful error message?

Reply:
If we attempt to add vrf to a member site, we just return the error whatever DCNM returns. DCNM returns "Invalid Fabric" as message in this scenario. The fabric details we fetch for the member site, does not specify that it is a member site of a multisite fabric. On creation this is just another easy fabric, only on moving this fabric to a multisite fabric, it becomes a member fabric. So only the multisite fabric details have the information on corresponding member site fabrics. And in this particular case if the user tries to configure a vrf in member site, it will be difficult to determine if it is a member site of a multisite fabric to generate a custom message in corresponding Ansible module.